### ISSUE TITLE
[BUG] use custom model and custom loss with eval

### ISSUE THREAD

**Issue by Cloudfimi** [slideflow/slideflow#449]
## Description

I get an error when I do a model evaluation with  custom model and custom loss.
## To Reproduce
1.P=sf.load_project('project/path')
2.dataset=P.dataset(tile_px=xxx, tile_um=xxx)
3.model='xxx.zip'
4.P.evaluate(model,outcomes='category',dataset=dataset)

The model uses a custom model structure and a custom loss during training, which is shown in the params.json file with model='custom' and loss='custom_classification '. 
Loading training results for inference on an external dataset results in error setting hyperparameter loss to custom_classification; unable to hyperparameters.error setting hyperparameter model to custom; unable to hyperparameters. and RuntimeError:Error(s) in loading state_dict for ModelWrapper: Missing key(s) in state_dict.
It is because the model auto-match ‘’xception‘’, I want to load my custom model structure how to do?

## Expected behavior

I want to load my custom model structure how to do? Can you please provide a clear description or a demo of the code? I would greatly appreciate your help.
## Environment:

 - Slideflow Version (e.g., 1.0):
 - OS (e.g., Ubuntu):
 - How you installed Slideflow (`pip`, source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:
